pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+line-length = 79
```

Comment:
Any particular reason not to use Black's default 88?

Their reasoning here:

* https://github.com/ambv/black#line-length